nginx.tmpl

```diff
@@ -198,6 +198,11 @@
 {{- define "upstream" }}
 upstream {{ .Upstream }} {
     {{- $server_found := false }}
+    {{- $loadbalance := first (keys (groupByLabel .Containers "com.github.nginx-proxy.nginx-proxy.loadbalance")) }}
+    {{- if $loadbalance }}
+    # Loadbalancing method: {{ $loadbalance }}
```

Comment:
I don't think it's useful to repeat the value of the `$loadbalance` variable here.

```suggestion
    # From the container's loadbalance label:
```